### ISSUE TITLE
cant upload folders, it stucks at %0

### ISSUE THREAD

**Issue by Eikosa** [mgilangjanuar/teledrive#324]
![image](https://user-images.githubusercontent.com/20538090/164087307-ff4b4b71-2278-460e-b9fd-90446b1a15f3.png)

![image](https://user-images.githubusercontent.com/20538090/164087353-319b3bce-cbde-4a95-b12c-51e4fe41726b.png)

**Comment by mgilangjanuar** [comment]
Is the file deleted/moved before uploading?

**Comment by Eikosa** [comment]
> Is the file deleted/moved before uploading?

no, it is a folder with file inside. i am using from this website
https://teledriveapp.herokuapp.com
